core/batch.go

```diff
@@ -160,156 +156,152 @@ func (sbbs *SimpleBatchBlockSender[I]) Flush() error {
 	batchState := sbbs.orchestrator.State()
 	sbbs.listMutex.Lock()
 	defer sbbs.listMutex.Unlock()
-	if err := sbbs.sender.SendList(batchState&SENDER, sbbs.list); err != nil {
+	if err := sbbs.sender.SendList(batchState&SOURCE, sbbs.list); err != nil {
 		return err
 	}
 	sbbs.list = sbbs.list[:0]
 
 	return nil
 }
 
-// BatchSendOrchestrator is an orchestrator for sending batches of blocks.
-type BatchSendOrchestrator struct {
+// BatchSourceOrchestrator is an orchestrator for sending batches of blocks.
+type BatchSourceOrchestrator struct {
 	flags util.SharedFlagSet[BatchState]
 	log   *zap.SugaredLogger
 }
 
-// NewBatchSendOrchestrator creates a new BatchSendOrchestrator.
-func NewBatchSendOrchestrator() *BatchSendOrchestrator {
-	return &BatchSendOrchestrator{
+// NewBatchSourceOrchestrator creates a new BatchSourceOrchestrator.
+func NewBatchSourceOrchestrator() *BatchSourceOrchestrator {
+	return &BatchSourceOrchestrator{
 		flags: *util.NewSharedFlagSet(BatchState(0)),
-		log:   log.With("component", "BatchSendOrchestrator"),
+		log:   log.With("component", "BatchSourceOrchestrator"),
 	}
 }
 
 // Notify notifies the orchestrator of a session event.
 // Events lead to state transitions in the orchestrator.
 //
-func (bso *BatchSendOrchestrator) Notify(event SessionEvent) error {
+func (bso *BatchSourceOrchestrator) Notify(event SessionEvent) error {
 	switch event {
-	case BEGIN_SESSION:
-		// bso.flags.Set(SENDER_READY) - now set by END_ENQUEUE
 	case BEGIN_SEND:
-		state := bso.flags.WaitAny(SENDER_READY|CANCELLED, 0)
+		state := bso.flags.WaitAny(SOURCE_PROCESSING|CANCELLED, 0)
```

Comment:
Just making sure... this was supposed to be a rename from READY to PROCESSING as well?  And all the other places this rename happened?

Reply:
Yes, the way it wound up being used, PROCESSING is more appropriate.